mne/io/tests/test_raw.py

```diff
@@ -556,9 +556,11 @@ def test_repr():
     """Test repr of Raw."""
     sfreq = 256
     info = create_info(3, sfreq)
-    r = repr(RawArray(np.zeros((3, 10 * sfreq)), info))
+    raw = RawArray(np.zeros((3, 10 * sfreq)), info)
+    r = repr(raw)
```

Comment:
This just testst `repr`, you need to call `raw._repr_html_()`

Reply:
this line is just to have a raw instance to call repr_html afterwards

Reply:
Whoops, missed that it's called below!